Framework/PythonInterface/plugins/algorithms/VisionReduction.py

```diff
@@ -7,6 +7,7 @@
 from mantid.simpleapi import *
 import csv
 import os
+import string
```

Comment:
You would need to disable using deprecated module string, and I would just import the functions that you need